source/_patterns/01-elements/link/link.scss

```diff
@@ -1,28 +1,34 @@
 @use "@db-ui/base/build/scss/variables" as *;
+@use "@db-ui/base/build/scss/scaling-placeholder" as *;
+@use "@db-ui/base/build/scss/color-placeholder" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
 @use "@db-ui/base/build/scss/icon/icons.helpers" as *;
 
 @import "../../../css/partials.meta";
-@import "link.variables";
 
 .elm-link {
-	border-radius: to-rem($pxValue: 4);
+	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
```

Comment:
@annam002 we even already have those two declarations from `https://db-ui.github.io/core/review/node_modules/@db-ui/base/build/scss/_init.scss` – so do we really need this one?
```scss
// Make sure every elements has the current color from container above
body {
	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
}

a {
	color: inherit;
}
```

Reply:
This might be relevant especially regarding scoped styles / ShadowDOM.